onnxconverter_common/float16.py

```diff
@@ -82,7 +82,33 @@ def make_value_info_from_tensor(tensor):
                          'FeatureVectorizer', 'Imputer', 'LabelEncoder', 'LinearClassifier', 'LinearRegressor',
                          'Normalizer', 'OneHotEncoder', 'SVMClassifier', 'SVMRegressor', 'Scaler',
                          'TreeEnsembleClassifier', 'TreeEnsembleRegressor', 'ZipMap', 'NonMaxSuppression', 'TopK',
-                         'RoiAlign', 'Resize', 'Range', 'CumSum', 'Min', 'Max', 'Upsample']
+                         'RoiAlign', 'Range', 'CumSum', 'Min', 'Max', 'Upsample']
+
+
+KEEP_ORIGINAL_DATA_TYPE_LIST = {
+    "Resize": 2,    # Resize operator, input[2] keep original type (tensor float)
+}
+
+
+# find all the constant input for specified op_type
+# e.g. search_node_input(model, "Resize", 2) will return all Resize.input[2] nodes
+def generate_attribute_blocklist(model):
+    inputs_constant = []
+    for op_type, input_index in KEEP_ORIGINAL_DATA_TYPE_LIST.items():
+        # find all the constant input[index] of operator by specified op_type
+        for node in model.graph.node:
+            if (node.op_type == op_type):
+                if len(node.input) > input_index:
```

Comment:
If len(node.input) <= input_index, does it mean the value in KEEP_ORIGINAL_DATA_TYPE_LIST is wrong and we need to provide some warning messages here to remind users?

Reply:
OK, we just remove this judge, if input_index is too big, then it will raise exception.